it/test/connectors/ReferenceDataConnectorSpec.scala

```diff
@@ -234,8 +245,8 @@ class ReferenceDataConnectorSpec extends ItSpecBase with ScalaCheckPropertyCheck
   "Reference Data" - {
 
     "getCustomsOfficesForCountry" - {
-      val countryCode = CountryCode("GB")
-      val url         = s"/$baseUrl/filtered-lists/CustomsOffices?data.countryId=${countryCode.code}&data.roles.role=DES"
+      val countryIds = Seq("GB", "XI")
+      val url        = s"/$baseUrl/lists/CustomsOffices?data.countryId=XI&data.countryId=GB&data.roles.role=DES"
```

Comment:
could this dynamically use the countryIds val above

Reply:
and the constants file

Reply:
Not going to make it dynamic as want to be sure that the url is being formed correctly with the query params included